README.md

```diff
@@ -17,9 +17,7 @@ def deps do
 end
 ```
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/stump](https://hexdocs.pm/stump).
+Documentation can be found at [https://hexdocs.pm/stump/Stump.html](https://hexdocs.pm/stump/Stump.html).
```

Comment:
are you planning to move the library under BBC before publishing it?

Reply:
I just published it last nigh so I could use it in the Fetcher just to have PR's ready, i'll move it and archive this and republish when it's working.